installers/msi-language/Deploy.wixproj

```diff
@@ -1,112 +1,113 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Why did this change completely? I can't review the changes in this file.

Reply:
I have no idea. I'll look into it first thing tomorrow, but in case he might offer a quicker answer... @mdrohmann If you have a quick answer for why this occurred, I'd appreciate knowing how to avoid it.

Reply:
This was a lined ending issue. It's been dos2unix-ed, and looks reasonable now.